src/launch/launcher.rs

```diff
@@ -1,12 +1,17 @@
 // SPDX-License-Identifier: Apache-2.0
 
+use super::Start;
+
 use crate::launch::linux::ioctl::*;
 use crate::launch::types::*;
 
 use std::io::Result;
 use std::os::unix::io::{AsRawFd, RawFd};
 
+pub struct Handle(u32);
```

Comment:
```suggestion
#[repr(transparent)]
pub struct Handle(u32);
```

Now you can use the `Handle` type directly in `LaunchStart`.